packages/sharing-smart-contract/.prettierrc

```diff
@@ -1,9 +1,9 @@
 {
-  "printWidth": 100,
+  "printWidth": 120,
```

Comment:
changing the print width made the PR much more difficult to read :grimacing: 
next time avoid doing this in a PR not dedicated to code formatting